@remirror/react/src/components/remirror-manager.tsx

```diff
@@ -27,6 +27,32 @@ export interface RemirrorManagerProps {
    * extension which determines the order in which it'll be loaded.
    *
    * By default these are placed after the JSX `RemirrorExtension`'s.
+   *
+   * @remarks
+   *
+   * If you have a `<RemirrorExtension />` element such as this:
+   *
+   * ```tsx
+   * const element = (
+   *   <RemirrorExtension
+   *     Constructor={Constructor}
+   *     priority={priority}
+   *     {...options}
+   *   />
+   * );
+   * ```
+   *
+   * you'd convert that to a valid entry in the `extensions` list like this:
+   *
+   * ```tsx
+   * const extensions = [
+   *   {
+   *     extension: new Constructor(options),
+   *     priority: priority || 2,
+   *   },
+   * ];
+   * ```
```

Comment:
![Screenshot_20200116_171211](https://user-images.githubusercontent.com/129910/72546845-67b4e100-3883-11ea-978b-c4f8d394f00a.png)

^ This I think will be very helpful for people :smiley: